OpenSim/Simulation/SimbodyEngine/SimbodyEngine.cpp

```diff
@@ -798,65 +798,51 @@ void SimbodyEngine::convertQuaternionsToDirectionCosines(double aQ1, double aQ2,
 //--- Private Utility Methods Below Here ---
 
 
-//_____________________________________________________________________________
-/**
- * Scale the dynamics engine
- *
- * @param aScaleSet the set of XYZ scale factors for the bodies
- * @param aFinalMass the mass that the scaled model should have
- * @param aPreserveMassDist whether or not the masses of the
- *        individual bodies should be scaled with the body scale factors.
- * @return Whether or not scaling was successful.
- */
-bool SimbodyEngine::scale(SimTK::State& s, const ScaleSet& aScaleSet, double aFinalMass, bool aPreserveMassDist)
+bool SimbodyEngine::scale(SimTK::State& s, const ScaleSet& scaleSet,
```

Comment:
Up to you @tkuchida (and maybe it is in your plan for phase II) but I think the this implementation could be moved into `Model` since the added indirection to the `SimbodyEngine` provides no advantages at all.

Reply:
Done. Also added note to CHANGELOG.md indicating that `SimbodyEngine::scale()` has been removed.